src/megaapi_impl.cpp

```diff
@@ -7968,6 +7976,11 @@ void MegaApiImpl::startUpload(const char *localPath, MegaNode *parent, int64_t m
 void MegaApiImpl::startUpload(const char* localPath, MegaNode* parent, const char* fileName, MegaTransferListener *listener)
 { return startUpload(false, localPath, parent, fileName, -1, 0, false, NULL, false, false, listener); }
 
+void MegaApiImpl::startUploadForSupport(const char *localPath, MegaTransferListener *listener)
+{
+    return startUpload(false, localPath, nullptr, nullptr, "supportdrop@mega.nz", -1, 0, false, nullptr, false, false, listener);
```

Comment:
Considering the relevance of this upload, perhaps we want to use the `startFirst = true`, indeed.
Also, it could be useful to give the ownership of the file to the SDK, so it deletes it after the upload, by setting the `isSourceFileTemporary = true`.
```suggestion
    return startUpload(true, localPath, nullptr, nullptr, "supportdrop@mega.nz", -1, 0, false, nullptr, true, false, listener);
```

Reply:
`startFirst` is a good idea. For FileTemporary, let's keep it false: we might want to upload something need to stay alive afterwards.